dev-requirements.txt

```diff
@@ -1,10 +1,18 @@
-pytest<3.3; python_version == '3.3'
-pytest>=4.6,<4.7; python_version != '3.3'
 coveralls
 flake8<3.6.0; python_version == '3.3'
 flake8>=3.7.0,<3.8.0; python_version != '3.3'
 flake8-coding
 flake8-future-import<0.4.6
+pytest<3.3; python_version == '3.3'
+pytest>=4.6,<4.7; python_version != '3.3'
+pytest-vcr==1.0.2; python_version != '3.3'
+pytest-vcr==0.3.0; python_version == '3.3'
+PyYAML<5.1; python_version == '3.3'
```

Comment:
I see what you did here. A way to get YAML into the project.

Reply:
LMAO, VCR just stores cassettes in YAML format. 😹